src/tests/__snapshots__/App.test.js.snap

```diff
@@ -1,9 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`App should render without crashing 1`] = `
-<div>
-  <h1>
-    Authors Haven
-  </h1>
-</div>
+<h1>
```

Comment:
Update the test file to reflect the name of the component you are testing for.

Reply:
@adesege the tests have been updated. The App component renders the Route component. The default route is the Landing page component. So I think this test is fine.